### PR TITLE
Fixing typo in task - certifcate changed to certificate

### PR DESCRIPTION
**Pull request by SteveMilnes**
## Changes

This work fixes as simple typo in the Receive grant payment certificate task in conversions.

Screenshot 2024-03-15 at 12 35 58 PM

Certifcate becomes `certificate`

## Checklist

- [x] Attach this pull request to the appropriate card in DevOps.
- [x] Update the `CHANGELOG.md` if needed.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +19,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ### Fixed
 
+- typo in Receive grant payment cert task action - certifcate to certificate
```
Comment (mec) [inline CHANGELOG.md:22]:
```suggestion
- typo in Receive grant payment certificate task action fixed - certifcate 
  becomes certificate
```

Comment (mec) [reply CHANGELOG.md:22]: You should be able to do this to silence the linter error - I think the line was just too long.

**Review by mec — APPROVED** [review]
It's not the end of the world, but I think you could just wrap your change log entry to silence the linter! 👍

**Comment by SteveMilnes** [comment]
Thanks @mec. I've committed your suggestions.

**Pull request merged**